x-pack/plugins/infra/public/pages/metrics/hosts/components/host_details_flyout/processes/processes.stories.tsx

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiCard } from '@elastic/eui';
+import { I18nProvider } from '@kbn/i18n-react';
+import type { Meta, Story } from '@storybook/react/types-6-0';
+import React from 'react';
+import { decorateWithGlobalStorybookThemeProviders } from '../../../../../../test_utils/use_global_storybook_theme';
+import { DecorateWithKibanaContext } from './preocesses.story_decorators';
+
+import { Processes, ProcessesProps } from './processes';
+
+export default {
+  title: 'infra/Host Details View/Processes',
```

Comment:
Perhaps `Processes tab` represents better this story?

Reply:
answered [here](https://github.com/elastic/kibana/pull/157200#discussion_r1192669389)